frontend/src/components/BaseDashboard/components/FilterColumn/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import "./FilterColumn.scss";
+import useOutsideClick from "../../../useOutsideClick";
+import { ReactComponent as FilterSvg } from "../../../../images/filter-icon.svg";
+
+const FilterColumn = ({ name, values, handleSelect, selectedValues }) => {
+  const [isVisible, setIsVisible] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const ref = useRef();
+  const uniqueValues = values.filter((x, i, a) => a.indexOf(x) === i);
```

Comment:
```setSearchResults(
      uniqueValues.filter(item =>
        item.toLowerCase().includes(searchTerm.toLowerCase())
      )
    );```

when user search second time it should search throw all values not into searchResult

Reply:
I see, you can do the filter inside that set, no need for a separate variable (performance might be better since now you're filtering on every render)